CoreDataAccessView/src/au/gov/asd/tac/constellation/views/dataaccess/plugins/clean/SplitNodesPlugin.java

```diff
@@ -94,6 +95,12 @@ public PluginParameters createParameters() {
         split.setStringValue(null);
         params.addParameter(split);
 
+        final PluginParameter<BooleanParameterValue> splitIntoSameLevelParameter = BooleanParameterType.build(SPLIT_INTO_SAME_LEVEL_PARAMETER_ID);
+        splitIntoSameLevelParameter.setName("Split Nodes into same level");
+        splitIntoSameLevelParameter.setDescription("Choose to split Nodes hanging off from the same parent node rather than from one of the new nodes");
```

Comment:
I'm wondering if there is a better name and description for this? I'm not sure whether this makes sense to a user without already knowing what it does. I can't think of anything off the top of my head though

Reply:
Agree, but I couldn't think of anything better either.

Reply:
I've looked over the code and still can't understand what this is supposed to be doing - could you explain it?

Reply:
Have you looked at the actual ticket #743. There are images too!